msal/application.py

```diff
@@ -69,8 +69,11 @@ def __init__(
                 {
                     "private_key": "...-----BEGIN PRIVATE KEY-----...",
                     "thumbprint": "A1B2C3D4E5F6...",
+                    "public_certificate": "...-----BEGIN CERTIFICATE-----..." (Only to be sent when using Subject Name Issuer Authentication)
                 }
 
+            public_certificate (optional) can be a public key certificate or certificate chain which is sent through
```

Comment:
Although our implementation here will actually support sending multiple certs, but since AAD works fine with only one cert, I incline to NOT mention the "or certificate chain" part at all, in order to discourage the app dev to actually configure a cert chain. The chain would still work, but the side effect is it would significantly increase the request payload size, which we better avoid.

By the way, if this paragraph can be limited to [roughly 80 chars per line](https://www.python.org/dev/peps/pep-0008/#maximum-line-length), that would be great! :-)